server/routes/service.js

```diff
@@ -1,3 +1,153 @@
-const transport = require('../routes/transport');
-const db = require('../db/db'); 
+require('dotenv').config();
+const bcrypt = require('bcrypt');
+const saltRounds = 10;
+const jwt = require('jsonwebtoken');
+const db = require('../db/db');
 
+
+// Check if the user is already authenticated
+function checkAuthStatus(authCookie) {
+  jwt.verify(authCookie, process.env.ACCESS_TOKEN_SECRET, (err) => {
+    if (err) {
+      console.log(err);
+      return '403';
+    }
+  });
+};
+
+// Check user existense
+async function signInDbQuery(name, pw) {
+  let checkedReturn;
+  const data = await db.checkIn(name);
+  if (data) {
+    const id = data.user_id;
+    const uName = data.user_name;
+    const err = data;
+    if (id && uName) {
+      await bcrypt.compare(pw, data.password)
+        .then(async (result) => {
```

Comment:
This function is defined as async but I don't see any `awaits` in the below code.  Is `async` really needed here?